addons/mail/static/src/new/discuss/channel_member_list.scss

```diff
@@ -0,0 +1,12 @@
+.o-mail-channel-member:hover {
+    background-color: map-get($grays, '300') !important;
+}
+
+.o-mail-channel-member-avatar-container {
+    width: 32px;
+    height: 32px;
+}
```

Comment:
Can use `$o-mail-thread-avatar-size` for consistent sizing (which is `36px` ATM)